rust/osm2lanes-web/src/main.rs

```diff
@@ -0,0 +1,164 @@
+use std::collections::BTreeMap;
+
+use serde::{Deserialize, Serialize};
+use web_sys::HtmlInputElement;
+use yew::prelude::*;
+
+use osm2lanes::{get_lane_specs_ltr, Config, Direction, DrivingSide, LaneSpec, LaneType};
+
+// Use `wee_alloc` as the global allocator.
```

Comment:
Is this a standard yew thing? Is there a reason to override the allocator for wasm? (I've only ever used `wasm-bindgen` before and never had to touch it)

Reply:
> wee_alloc is a tiny allocator that is much smaller than the allocator that is normally used in Rust binaries. Replacing the default allocator with this one will result in smaller Wasm file sizes, at the expense of speed and memory overhead.

> The slower speed and memory overhead are minor in comparison to the size gains made by not including the default allocator. This smaller file size means that your page will load faster, and so it is generally recommended that you use this allocator over the default, unless your app is doing some allocation-heavy work.

Reply:
https://yew.rs/docs/next/advanced-topics/optimizations